src/Controller/Component/TagsComponent.php

```diff
@@ -50,19 +52,43 @@ public function initialize(array $config): void
      */
     public function load(): Query
```

Comment:
```suggestion
    public function load(string|null $lang = null): Query
```
and the phpdoc (github does not make me add a suggestion on those lines):
```
* @param string|null $lang Language code to use.
```